tox.ini

```diff
@@ -78,6 +78,14 @@ changedir = {toxinidir}/tasks
 extras =
 commands = python update_embedded.py
 
+[testenv:zipapp]
```

Comment:
definitely add it to the default targets too 👍 just to ensure the tox interface does not break